Conceptpower+Spring/pom.xml

```diff
@@ -1,800 +1,783 @@
-				<artifactId>buildnumber-maven-plugin</artifactId>
-				<version>1.4</version>
-				<executions>
-					<execution>
-						<phase>initialize</phase>
-						<goals>
-							<goal>create</goal>
-						</goals>
-					</execution>
-				</executions>
-				<configuration>
-					<doCheck>false</doCheck>
-					<doUpdate>false</doUpdate>
-					<revisionOnScmFailure></revisionOnScmFailure>
-					<items>
-						<item>buildNumber\d*</item>
-						<item>timestamp</item>
-					</items>
-					<format>#{0} ({1,date,yyyy-MM-dd HH:mm:ss})</format>
-				</configuration>
-			</plugin>
-
-			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>build-helper-maven-plugin</artifactId>
-				<version>3.0.0</version>
-				<executions>
-					<!-- Add a new source directory to our build -->
-					<execution>
-						<id>add-integration-test-sources</id>
-						<phase>generate-test-sources</phase>
-						<goals>
-							<goal>add-test-source</goal>
-						</goals>
-						<configuration>
-							<!-- Configures the source directory of our integration tests -->
-							<sources>
-								<source>src/integration-test/java</source>
-							</sources>
-						</configuration>
-					</execution>
-					<!-- Add a new resource directory to our build -->
-					<execution>
-						<id>add-integration-test-resources</id>
-						<phase>generate-test-resources</phase>
-						<goals>
-							<goal>add-test-resource</goal>
-						</goals>
-						<configuration>
-							<!-- Configures the resource directory of our integration tests -->
-							<resources>
-								<resource>
-									<filtering>true</filtering>
-									<directory>src/integration-test/resources</directory>
-								</resource>
-							</resources>
-						</configuration>
-					</execution>
-
-				</executions>
-			</plugin>
-           
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-surefire-plugin</artifactId>
-				<version>3.0.0-M3</version>
-				<configuration>
-					<!-- Skips unit tests if the value of skip.unit.tests property is true -->
-					<skipTests>${skip.unit.tests}</skipTests>
-					<!-- Excludes integration tests when unit tests are run -->
-					<excludes>
-						<exclude>**/*IT.java</exclude>
-					</excludes>
-				</configuration>
-			</plugin>
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-failsafe-plugin</artifactId>
-				<version>3.0.0-M3</version>
-				<executions>
-					<!-- Invokes both the integration-test and the verify goals of the Failsafe 
+      <debug_level>debug</debug_level>
+      <skip.integration.tests>false</skip.integration.tests>
+      <skip.unit.tests>false</skip.unit.tests>
+      <concept.list.url>https://www.dropbox.com/s/r82nymnt2247zz4/conceptLists.db?raw=1</concept.list.url>
+      <concept.types.url>https://www.dropbox.com/s/nwn15s84fzz37s5/conceptTypes.db?raw=1</concept.types.url>
+      <users.url>https://www.dropbox.com/s/p6h3974hmlouq0i/users.db?raw=1</users.url>
```

Comment:
we gotta fix the integration tests; I think for now I would add test database configs and fill the database before running the test cases and empty it afterwards. Not ideal but will probably work.